Pin ipython<8.13 for python 3.8

smackesey opened the pull request:
## Summary & Motivation

## How I Tested These Changes

smackesey commented:
Current dependencies on/for this PR:
* master
  * **PR #13998** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13998" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13998?utm_source=stack-comment).

alangenfeld reviewed (approved):
link the ipython issue maybe

smackesey commented:
@alangenfeld linked, I also updated a `dagster-ge` snapshot. Not sure why exactly it started failing but it was a snapshot of a notebook, I presume some subsidiary lib of jupyter (which we're not pinning) was also updated and changed the formatting.
